content/en/docs/developer-documentation/build-install-container.md

```diff
@@ -38,6 +38,16 @@ Build the MicroShift AIO container:
 ```Bash
 make microshift-aio
 ```
+## Tagging the Image
+
+It is necessary to tag the image as latest to ensure that the newly created image is used when starting MicroShift with systemd.
+
+```Bash
+IMAGE=$(podman images | grep micro | awk '{print $3}')
+podman tag ${IMAGE} quay.io/microshift/microshift:latest
+```
+
```

Comment:
this is assuming the image build happens on same same machine where microshift runs. I usually build locally, push to quay.io/sallyom then change the unit file to match. I'm usually launching in a VM - but otoh, the aio is meant to run easily on your local machine. With podman auto-update, will podman first look in local images for a new digest? or, will podman reach out to quay.io/microshift first to find the :latest digest there? ah, looks like it depends on what's configured (in the unit file): https://docs.podman.io/en/latest/markdown/podman-auto-update.1.html  - [we set to "registry"](https://github.com/redhat-et/microshift/blob/main/packaging/systemd/microshift-aio.service#L18) so it won't look in local storage (`--label io.containers.autoupdate=registry`) To further drive home the point that aio is meant for local development, we _could_ change that to "local". Anyways, for this PR, might make more sense to tag the image w/ whatever, then, instruct to [modify the unit file]((https://github.com/redhat-et/microshift/blob/main/packaging/systemd/microshift-aio.service) to match.

Reply:
oh jeez, we aren't mentioning the unit file for AIO at all, yet, so this needs to work for both straight podman and systemd.

How about, for now, something like:
`If building & pushing to a remote registry such as quay.io/yourname, and when running with systemd, be sure the tag of the newly built image matches what is listed in the unit file at /etc/system/systemd/crio.service`